### DISCUSSION TITLE
Si può avere la classe che frequenta?

### DISCUSSION THREAD

**Discussion by caamillo** [47PADO47/Classeviva.js#11] [Ideas]
Utilizzando esempio il metodo .getCard() o .getCards() non trovo la classe frequentante.
Riesco comunque a trovarla esempio nelle ultime lezioni frequentate, esiste un metodo che fa return della classe?

**Comment by 47PADO47** [comment]
Usa `Rest#getSubjects()` per le varie materie, oppure protresti provare con  `Web#getRubrica()` e guardare nell'array `classi`, ma potrebbe essere vuoto in quanto recentemente avevano disattivato la possibilità di messaggiare con account studente.

**Comment by Giasinit** [comment]
> Utilizzando esempio il metodo .getCard() o .getCards() non trovo la classe frequentante. Riesco comunque a trovarla esempio nelle ultime lezioni frequentate, esiste un metodo che fa return della classe?

Ciao, io ho trovato un metodo: basta prendere l'agenda tramite exportXmlAgenda e scaricarla, poi all'interno si può trovare la classe di referenza.
se vuoi ho il codice mio sviluppato in PHP per vedere la classe:
```
<?php
$agendastudente = simplexml_load_file('agenda.xml');
echo 'Classe Frequentata: '.$agendastudente->Worksheet->Table ->Row[1]->Cell[8]->Data.'<br/>';
?>  
```
Questo va a vedere il file agenda.xml e poi va a mostrare la classe.
questo è utile anche per vedere i compiti e le verifiche assegnate.
spero ti sia utile! ciao.
